nodejs/packages/layer/install-externals.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -euf -o pipefail
+
+rm -rf ./build/workspace/node_modules
+
+# Space separated list of external NPM packages
+EXTERNAL_PACKAGES=( "import-in-the-middle" )
```

Comment:
js newbie here :) 

Instead of having this separated script file with a list of external dependencies, would it work if we had a separated node package with package.json where you define all the dependencies. This would be a "noop package", but it would allow you to explicitly select what version of the external dependencies you want to use. and then you install it with just `npm install --prefix ./build/workspace --production --ignore-scripts`.

Reply:
The reason I have used bash script to install external packages is that I want to use the same version of `import-in-the-middle` coming from OTEL. So, I have to resolve its version dynamically. That is why I first resolve its version by npm query below and then install it with that specific version.